setup.cfg

```diff
@@ -1,7 +1,5 @@
 [tool:pytest]
-addopts = --ignore=setup.py --doctest-modules --cov-report term-missing --cov .
```

Comment:
Why is this not necessary anymore? With this we can get the reports and coverage.

Reply:
Please add it back :)

Reply:
Thanks, I was able to figure out the error message I was getting on this one 😄